From 00000000000000000000000000000000000000000000 Sep 17 00:00:00 2001
From: Alexander Kabaev <kan@fb.com>
Date: Wed, 22 May 2019 16:04:03 -0400
Subject: [PATCH] Bump the number of pre-created dpmcp objects.

Each of dpmac, dpio, dprtc and dpni needs one. We have
8 dpios, 10 dpmacs, and 2 dpni's, which require 20 dpmcp
objects, create 21 for the good measure.
---
 config/ls1088a/RDB/dpl-eth.0x1D_0x0D.dts | 18 +++++++++++++++++-
 1 file changed, 17 insertions(+), 1 deletion(-)

diff --git a/config/ls1088a/RDB/dpl-eth.0x1D_0x0D.dts b/config/ls1088a/RDB/dpl-eth.0x1D_0x0D.dts
index 58e2762..20ae520 100644
--- a/config/ls1088a/RDB/dpl-eth.0x1D_0x0D.dts
+++ b/config/ls1088a/RDB/dpl-eth.0x1D_0x0D.dts
@@ -69,7 +69,7 @@
 				/* ------------ DPMCPs --------------*/
 				obj_set@dpmcp {
 					type = "dpmcp";
-					ids = <1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17>;
+					ids = <1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25>;
 				};
 
 				/* ------------ DPCON --------------*/
@@ -193,6 +193,22 @@
 		};
 		dpmcp@17 {
 		};
+		dpmcp@18 {
+		};
+		dpmcp@19 {
+		};
+		dpmcp@20 {
+		};
+		dpmcp@21 {
+		};
+		dpmcp@22 {
+		};
+		dpmcp@23 {
+		};
+		dpmcp@24 {
+		};
+		dpmcp@25 {
+		};
 
 		/* ------------ DPCON --------------*/
 		dpcon@1 {
-- 
2.30.2

